samples/app.py

```diff
@@ -0,0 +1,249 @@
+                                                                                execute_mboxes=execute_mboxes)
+
+    response = Response(target_delivery_response.get("response").to_str(), status=200, mimetype='application/json')
+    set_target_cookies(response, target_delivery_response)
+    return response
+
+
+@app.route('/prefetchMbox', methods=['GET'])
+def prefetch_mbox():
+    """
+    Make a single request that includes both execute mboxes request as well as prefetch mboxes request.
+    Get cookies for request if they exist, and on response set Target cookie for use with next request
+    """
+    get_offers_options = initialize_options(request)
+
+    mboxes = request.args.get("mbox", "")
+    prefetch = request.args.get("prefetch", "")
+    if not mboxes and not prefetch:
+        return Response("No Content", status=204)
+
+    execute_mboxes = [MboxRequest(name=mbox, index=index) for index, mbox in enumerate(mboxes.split(","))] \
+        if mboxes else None
+    prefetch_mboxes = [MboxRequest(name=mbox, index=index) for index, mbox in enumerate(prefetch.split(","))] \
+        if prefetch else None
+    target_delivery_response = target_service.get_mbox_target_delivery_response(request, get_offers_options,
+                                                                                execute_mboxes=execute_mboxes,
+                                                                                prefetch_mboxes=prefetch_mboxes)
+
+    response = Response(target_delivery_response.get("response").to_str(), status=200, mimetype='application/json')
+    set_target_cookies(response, target_delivery_response)
+    return response
+
+
+def get_offers_callback(request_id, target_delivery_response):
+    response = Response(target_delivery_response.get("response").to_str(), status=200, mimetype='application/json')
+    set_target_cookies(response, target_delivery_response)
+    results[request_id] = response
+
+
+def get_offers_err_callback(request_id, error):
+    response = Response("Exception during get_offers call: {}".format(error), status=500, mimetype='application/json')
+    results[request_id] = response
+
+
+def get_target_delivery_response_async_callback(get_offers_options, request_id, views):
+    get_offers_options["callback"] = functools.partial(get_offers_callback, request_id)
+    get_offers_options["err_callback"] = functools.partial(get_offers_err_callback, request_id)
+
+    async_result = target_service.prefetch_views_target_delivery_response(request,
+                                                                          request_id,
+                                                                          get_offers_options,
+                                                                          views)
+    async_result.wait()  # This is blocking, pass AsyncResult into an event loop
+    return results.pop(request_id)
+
+
+async def get_target_delivery_response_async_await(get_offers_options, request_id, views):
```

Comment:
```suggestion
async def get_target_delivery_response_async(get_offers_options, request_id, views):
```

Reply:
I had named this fn `async_await` because there are actually 2 different async options supported (callbacks for python 2, async/await for python 3).  So I wanted it to be clear that this particular fn goes with the async/await example.  But I can change it if you think it's confusing...